packages/migrate/src/migrate.ts

```diff
@@ -62,14 +62,16 @@ export default async function migrate(input: MigrateInput): Promise<MigrateOutpu
   for (const fileName of fileNames) {
     logger?.info(`Processing file: ${fileName}`);
 
+    let allChangedFiles: Set<string> = new Set();
+
     for (const pluginClass of plugins) {
       const plugin = new pluginClass(service, logger, reporter);
-      const updatedText = await plugin.run({ fileName });
-      service.setFileText(fileName, updatedText);
+      const changedFiles = await plugin.run({ fileName });
```

Comment:
Again here please confirm this no longer needs to be passed though the service

Reply:
same as above